sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/query/DistinctHashTest.java

```diff
@@ -0,0 +1,151 @@
+package com.azure.cosmos.implementation.query;
+
+import com.azure.cosmos.implementation.routing.UInt128;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DistinctHashTest {
+
+    @Test(groups = {"unit"})
+    public void TestHashForNull() throws IOException {
```

Comment:
to be consistent with existing tests:

drop the "Test" prefix. here and other new tests..

Reply:
changed to *Hash